### DISCUSSION TITLE
模板消息群发

### DISCUSSION THREAD

**Discussion by 253287742** [w7corp/easywechat#2421] [Q&A]
使用  "overtrue/wechat": "~5.0",
使用循环 发送超过五百条 模板消息
$app->template_message->send($data)
执行到三百条的时候会提示502超时，请问我该如何优化？

**Comment by overtrue** [comment]
这属于 PHP 基础知识，批量任务一般使用队列方式执行，而不是一个进程发很久。

**Reply by 253287742** [reply]
好的 确实是新手问题了 研究完队列 完美解决了  Thanks♪(･ω･)ﾉ
